app/routes/application.js

```diff
@@ -0,0 +1,22 @@
+import EmpressApplicationRoute from 'empress-blog/routes/application';
+import { inject as service } from '@ember/service';
+
+
+export default class ApplicationRoute extends EmpressApplicationRoute {
+  @service metrics;
+  @service router;
+
+  constructor() {
+    super(...arguments);
+
+    this.router.on('routeDidChange', () => {
+      const page = this.router.currentURL;
+      const title = this.router.currentRouteName ?? 'unknown';
```

Comment:
**question:**

Will there be a case when the `router` service doesn't know `currentRouteName`?